development_scripts/demo_deconvolution.py

```diff
@@ -137,11 +143,11 @@
 
 # now let's grab the deconvoluted current for a tspan of interest
 # first, need to calibrate the measurement, we'll use siq here
-ca_dark_day1.set_siq_quantifier(calibration=Calibration(cal_list=[F_o2]), carrier="He")
+ca_dark_day1.add_calculator(calibration)
 # now let's deconvolute
-t_deconvoluted, v_deconvoluted = ca_dark_day1.grab_deconvoluted_signal(
+t_deconvoluted, v_deconvoluted = imp_resp_model.grab_deconvoluted_signal(
```

Comment:
I think that calculators should be allowed to have whatever quantification methods are useful, not just `calculate_series` (which is what `Measurement` uses), but I agree the name `grab` should be reserved for the method of `Measurement` to avoid convusion. I've renamed this method `calc_deconvoluted_signal`.